test/functional/apps/visualize/_tsvb_markdown.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from 'expect.js';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+// tslint:disable-next-line:no-default-export
+export default function({ getPageObjects }: FtrProviderContext) {
+  const { visualBuilder, timePicker } = getPageObjects([
+    'visualBuilder',
+    'timePicker',
+    'visualize',
+  ]);
+
+  describe('visual builder', function describeIndexTests() {
+    describe('markdown', () => {
+      before(async () => {
+        await visualBuilder.resetPage();
+        await visualBuilder.clickMarkdown();
+        await timePicker.setAbsoluteRange('2015-09-22 06:00:00.000', '2015-09-22 11:00:00.000');
+      });
+
+      afterEach(async () => {
+        await visualBuilder.clearMarkdown();
+      });
+
```

Comment:
`enterMarkdown` does the cleaning before typing new value. Redundant `afterEach`. If `clearMarkdown` is flaky, let's fix it, not duplicate calls.